README.md

```diff
@@ -143,21 +144,50 @@ docker run -it --rm \
   -w $(realpath "${PWD}") \
   --network host \
   vagrantlibvirt/vagrant-libvirt:latest \
-    vagrant status
+  vagrant status
```

Comment:
I have a slight preference to keep the indent formatting for this and the commands below as:
```bash
<container-cli> \
  <arg-to-container-cli> \
    <arg-to-container-entrypoint>
```

As a subtle hint that the line `vagrant status` isn't being passed to the docker command, but rather what ever docker itself is executing, since I think a note is needed below on the change to the mount path as it might be useful information if rootless docker behaves similar to podman.